gulp-file.ts

```diff
@@ -100,7 +101,14 @@ gulp.task('build:appsrc', ['lint', 'clean'], () => {
     return build(files.site.ts, paths.site.app);
 });
 
-gulp.task('build', ['build:tests', 'build:src', 'build:appsrc']);
+// Compiles LESS > CSS
+gulp.task('build:less', function(){
+    return gulp.src(paths.site.src + 'styles.less')
+        .pipe(less())
+        .pipe(gulp.dest(paths.site.app + '/css'));
+});
+
```

Comment:
() => { // }; :)

You can declare less inside the task itself, and it might be nice to remove the '/css' hard coding, and include the trailing '/'.